lib/remoteproc/remoteproc.c

```diff
@@ -283,6 +284,8 @@ int remoteproc_shutdown(struct remoteproc *rproc)
 					rproc->state = RPROC_OFFLINE;
 				}
 			}
+
+			rproc->bitmap = 0;
```

Comment:
Any reason to do it again here?  It should already be set to 0 in `remoteproc_stop()`, right?

Reply:
Agree with @arnopo.

Reply:
Is there a reasonable use case where remoteproc_shutdown() is called without then calling remoteproc_stop()?

Reply:
@arnopo @tnmysh @edmooring Hi, yes, in our case, we only call `remoteproc_shutdown()` because `remoteproc_shutdown()` will call the stop too, and if there is no use case where only call `remoteproc_stop()`, I can delete the bitmap clear in `remoteproc_stop()`.

Reply:
seems that the call stack to update the `rproc->bitmap` is 
```
remoteproc_load
  remoteproc_get_rsc_table
    handle_rsc_table
      handle_vdev_rsc
        remoteproc_allocate_id
```
so the bitmap is set on load and should be reset on stop.

Reply:
Got it, Done.